docs/source/get_started/kedro_concepts.md

```diff
@@ -60,7 +60,7 @@ project-dir         # Parent directory of the template
 ├── docs            # Project documentation
 ├── logs            # Project output logs (not committed to version control)
 ├── notebooks       # Project-related Jupyter notebooks (can be used for experimental code before moving the code to src)
-├── pyproject.toml  # Identifies the project root and [contains configuration information](../faq/architecture_overview.md#kedro-project)
+├── pyproject.toml  # Identifies the project root and contains configuration information
```

Comment:
See above about linking to project settings page

Reply:
Same comment in return 😆  -- I'm not seeing the `.toml` project documentation at present, only a passing mention to it in the (now defunct) architecture page. When that is added, we should definitely add some links to it.